src/Compilers/Core/Portable/SourceGeneration/Nodes/SyntaxTransformNode.cs

```diff
@@ -0,0 +1,118 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections.Generic;
+using System.Collections.Immutable;
+using System.Text;
+using System.Threading;
+using System.Linq;
+
+namespace Microsoft.CodeAnalysis
+{
+    internal interface ISyntaxTransformNode
+    {
+        ISyntaxTransformBuilder GetBuilder(DriverStateTable previousStateTable);
```

Comment:
```suggestion
        ISyntaxTransformBuilder GetBuilder(DriverStateTable previousTable);
```

Most of the other code uses this name